src/lotus_json/mod.rs

```diff
@@ -289,12 +305,70 @@ pub mod base64_standard {
     }
 }
 
+/// MUST NOT be used in any `LotusJson` structs
```

Comment:
I guess without reflection we won't be able to ensure this?

Reply:
Yep, or a custom linter